core-bundle/src/Resources/contao/library/Contao/System.php

```diff
@@ -188,12 +188,12 @@ protected function import($strClass, $strKey=null, $blnForce=false)
 			{
 				$this->arrObjects[$strKey] = $container->get($strClass);
 			}
-			elseif (($container->getParameter('kernel.debug') || !\class_exists($strClass)) && self::isServiceInlined($strClass))
+			elseif (($container->getParameter('kernel.debug') || !class_exists($strClass)) && self::isServiceInlined($strClass))
 			{
 				// In debug mode, we check for inlined services before trying to create a new instance of the class
 				throw new ServiceNotFoundException($strClass, null, null, array(), sprintf('The "%s" service or alias has been removed or inlined when the container was compiled. You should either make it public, or stop using the container directly and use dependency injection instead.', $strClass));
 			}
-			elseif (!\class_exists($strClass))
+			elseif (!class_exists($strClass))
```

Comment:
Why are we not using the global functions here? We should use it everywhere, also in legacy code because it's a huge performance gain.

Reply:
because the CS fixer asked me not to … 😢

Reply:
@leofeyer any reason why we do not use it? It's wasted performance :(

Reply:
>  It's wasted performance

Is it really? This will only shave off a small amount of time (in the ns range) for the function call which will only add up to a significant amount (ms range) if it happens millions of times.

But prefixing also doesn't hurt, so +1 for a homogenous coding style (imo not necessary for the legacy code).

Reply:
Loads of this stuff could be replaced by OPCodes. If it was added to the whole code base, I'm pretty sure it would be measurable.

Reply:
`class_exists()` is not one of the compiler optimizable functions: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php#L355-L400